firedrake/interpolation.py

```diff
@@ -480,6 +495,124 @@ def compose_map_and_cache(map1, map2):
     return cmap
 
 
+def vom_cell_parent_node_map_extruded(vertex_only_mesh, extruded_cell_node_map):
+    """Build a map from the cells of a vertex only mesh to the nodes of the
+    source mesh's cell node map where the source mesh is extruded.
+
+    Parameters
+    ----------
+    vertex_only_mesh : :class:`mesh.MeshGeometry`
+        The ``mesh.VertexOnlyMesh`` whose cells we iterate over.
+    extruded_cell_node_map : :class:`PyOP2.Map`
+        The cell node map of the function space on the extruded mesh within
+        which the ``mesh.VertexOnlyMesh`` is immersed.
+
+    Returns
+    -------
+    :class:`PyOP2.Map`
```

Comment:
It's the same as in the existing `compose_map_and_cache` function but I notice elsewhere it's 
```
:class:`pyop2.Map`
```
I don't think the API documentation builds for these functions anyway since they aren't included in `__all__` at the top of the file.

Reply:
I'll rebase and change to 
```
:class:`pyop2.Map`
```
anyway